Mathlib/MeasureTheory/Integral/Periodic.lean

```diff
@@ -283,8 +334,8 @@ section RealValued
 
 open Filter
 
-variable {g : ℝ → ℝ}
-variable (hg : Periodic g T) (h_int : ∀ t₁ t₂, IntervalIntegrable g MeasureSpace.volume t₁ t₂)
+variable {g : ℝ → ℝ} {t₁ : ℝ}
+variable (hg : Periodic g T) (h_int : IntervalIntegrable g MeasureSpace.volume 0 T)
```

Comment:
One final nitpick: the assumption `h_int` is definitely needed in the next two lemmas, but I think it can be removed from the other ones, as positivity of the integral implies that the function is integrable (by definition, we set the integral of non-integrable functions to be zero). So, you could use something like
```
  have h_int : IntervalIntegrable g MeasureSpace.volume 0 T := 
    intervalIntegrable_of_integral_ne_zero h₀.ne'
```
at the beginning of the proof of the lemmas `tendsto_atTop_intervalIntegral_of_pos` and `tendsto_atBot_intervalIntegral_of_pos` instead of assuming `h_int`.

To implement this, it's probably best to remove `h_int` from the `variable` line, and write it down explicitely in the two lemmas that need it.

Reply:
Allright. Will do.

Reply:
Implemented as requested.